app/views/admin/users/_current_roles.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="crayons-card p-6 grid gap-2">
+  <h2>Current Roles</h2>
```

Comment:
I decided to pull "Current Roles" out into its own partial to make it more extensible and because with the addition of the removal button to roles, it no longer makes sense to show "Current Roles" on the `/admin/users/:id` page.